backend/app/services/share/knowledge_share_service.py

```
@@ -194,7 +204,46 @@ def _get_resource(
             )
             return kb
 
-        logger.warning(f"[_get_resource] User has NO explicit shared access")
+        logger.info(f"[_get_resource] User has NO explicit shared access")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)